deployments/jwebbinar/image/Dockerfile.custom

```diff
@@ -43,6 +43,17 @@ RUN cd /tmp && \
     make && \
     make install
 
+# Download and unzip PDRs4All dataset from Box
```

Comment:
Is this a large amount of data? It might not be right to put this data in the image if so - better to pre-load it into EFS and get it mounted in the PVC